doc/rst/02_getting_started.rst

```diff
@@ -0,0 +1,161 @@
+
+.. literalinclude:: ../../examples/capture_rgb/src/mapfile.xc
+   :language: c
+   :start-at: int main(void)
+   :end-at: return 0;
+
+
+- ``user_app.c``: This contains the user_app thread, which is responsible for setting the buffer where the image data will be stored and for processing the image data after it has been captured. Image size and properties are user centric, meaning that the user can set the image size and properties according to their needs. The library will then handle the conversion of the image data to the desired format.
+
+.. literalinclude:: ../../examples/capture_rgb/src/user_app.c
+   :language: c
+   :start-at:  // Image and configuration
+   :end-before: save_image(&image, FILE1_NAME);
+
+First, the user must define the objects ``camera_cfg_t`` and ``image_t``. The ``camera_cfg_t`` struct contains the camera configuration parameters, such as offsets to define the region of interest and the acquisition mode. The ``image_t`` struct contains the image data and properties, such as width, height, a pointer to the image buffer, and a pointer to the previously declared configuration. In the example, the user expects an image of 200x200x3 RGB int8 image. 
+
+Next, the user must call the ``camera_isp_coordinates_compute()`` function, which takes a pointer to ``image_t``. This call computes the coordinates of the region of interest (ROI) based on the camera configuration (``camera_cfg_t``) and the capture mode. In this case, it will be captured in RGB format and downsampled by a factor of 2 and starting at position (0,0).
+
+Finally, the user must call the ``camera_isp_start_capture()`` function to start the camera capture process. When the user needs the frame, they can call the ``camera_isp_get_capture()`` function. This function will block until the frame is ready.
+
+Once the frame is ready, the user can process the image data and save it to a file using the ``save_image()`` function.
+
+The following sections provide detailed information about the requirements for running the example, as well as step-by-step instructions on how to build and execute it.
+
+.. _Build and Run the Application:
+
+Hardware Requirements
+---------------------
+
+The library is designed to work with the following hardware:
+
+- |vision board| (|vision board ref|)
+- 2x Micro USB cable (Power supply and xTag)
+
+Hardware Setup
+--------------
+
+- Plug the 1x Micro USB cable to both the host computer and the ``DEBUG`` Micro USB port.
```

Comment:
Not sure if it's an issue but we first say 2 cables and only use one in the photo